semver/_shared.ts

```diff
@@ -71,13 +71,13 @@ export function compareIdentifier(
 // ## Numeric Identifier
 // A single `0`, or a non-zero digit followed by zero or more digits.
 
-const NUMERICIDENTIFIER = "0|[1-9]\\d*";
+const NUMERICIDENTIFIER = `0|[1-9]\\d*`;
```

Comment:
```suggestion
const NUMERIC_IDENTIFIER = `0|[1-9]\\d*`;
```
I think we should do the same with the other constants (if they're private) to make them more readable.

Reply:
I agree, I was going to update the names in a separate PR. Do you want to do it in this PR?

Reply:
Ah, actually, doing it in another PR is a good idea. It might be worth checking the rest of the sub-module for similar fix opportunities.